Packs/Use_Case_Builder/IncidentFields/incidentfield-Use_Case_TIM_Best_Practice.json

```
@@ -20,7 +20,7 @@
     "required": false,
     "sla": 0,
     "system": false,
-    "template": "# Threat Intel Management\n## Manually Add Indicators to the Exclusion List\nFrom the Exclusion List page, you can manually add a single indicator or define indicators using a regular expression (regex) or CIDR.\n\n### Regex\n\nA regular expression enables you to identify a sequence of characters in an unknown string. The following example would identify www.demisto.com: [A-Za-z0-9!@#$%\\.\u0026]*demisto[A-Za-z0-9!@#$%\\.\u0026]*.\n\n### CIDR\nClassless inter-domain routing (CIDR) enables you to define a range of IP addresses. For example, the IPv4 block 192.168.100.0/22 represents the 1024 IPv4 addresses from 192.168.100.0 to 192.168.103.255.\n\n## Indicator Management: [TIM](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.5/Cortex-XSOAR-Threat-Intel-Management-Guide/Exclusion-List)",
+    "template": "# Threat Intel Management\n## Manually Add Indicators to the Exclusion List\nFrom the Exclusion List page, you can manually add a single indicator or define indicators using a regular expression (regex) or CIDR.\n\n### Regex\n\nA regular expression enables you to identify a sequence of characters in an unknown string. The following example would identify www.demisto.com: [A-Za-z0-9!@#$%\\.\u0026]*demisto[A-Za-z0-9!@#$%\\.\u0026]*.\n\n### CIDR\nClassless inter-domain routing (CIDR) enables you to define a range of IP addresses. For example, the IPv4 block 192.168.100.0/22 represents the 1024 IPv4 addresses from 192.168.100.0 to 192.168.103.255.\n\n## Indicator Management: [TIM (XSOAR 6.13)](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.13/Cortex-XSOAR-Threat-Intel-Management-Guide/Exclusion-List) or [TIM (XSOAR 8)](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Delete-and-exclude-indicators)",
```

```suggestion
    "template": "# Threat Intel Management\n## Manually Add Indicators to the Exclusion List\nFrom the Exclusion List page, you can manually add a single indicator or define indicators using a regular expression (regex) or CIDR.\n\n### Regex\n\nA regular expression enables you to identify a sequence of characters in an unknown string. The following example would identify www.demisto.com: [A-Za-z0-9!@#$%\\.\u0026]*demisto[A-Za-z0-9!@#$%\\.\u0026]*.\n\n### CIDR\nClassless inter-domain routing (CIDR) enables you to define a range of IP addresses. For example, the IPv4 block 192.168.100.0/22 represents the 1024 IPv4 addresses from 192.168.100.0 to 192.168.103.255.\n\n## Indicator Management: [TIM (XSOAR 6.13)](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.13/Cortex-XSOAR-Threat-Intel-Management-Guide/Exclusion-List) or [TIM (XSOAR 8 Cloud)](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Delete-and-exclude-indicators)",
```
What about on-prem? https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Delete-and-exclude-indicators